infer/src/base/CommandDoc.ml

```diff
@@ -57,6 +57,13 @@ $(b,infer) $(i,[options])|}
     ~description:[`P "Analyze the files captured in the project results directory and report."]
     ~see_also:InferCommand.[Report; Run]
 
+let analyze_json = 
+  mk_command_doc ~title:"Infer Analysis" ~short_description:"analyze the files captured by infer"
+    ~synopsis:{|$(b,infer) $(b,analyze) $(i,[options])
+$(b,infer) $(i,[options])|}
+    ~description:[`P "Analyze the files captured in the project results directory and report."]
+    ~see_also:InferCommand.[Report; Run]
```

Comment:
Could you update this to document the new command? Having an example of how to run the new command would be helpful. Would you be able to paste an example json somewhere? I'd love to be able to experiment with this PR locally.

Reply:
> Could you update this to document the new command? Having an example of how to run the new command would be helpful. Would you be able to paste an example json somewhere? I'd love to be able to experiment with this PR locally.

Added. And I have added example jsons in examples/dotnet_hello/jsons